Clinical-Genomics/cgweb
# Require elution buffer for mip exome and panel applications

This PR adds a fix for the missing elution buffers from Orderportal

How to test:
install branch on clinical-db
install https://github.com/Clinical-Genomics/cg/pull/527 on clinical-db
create fastq order with PA/EX app
submit

Expected result:
- [x] elution buffer should be required when PAx/EXx applications

- [x] code CR
- [x] test @patrikgrenfeldt 
- [x] deploy @patrikgrenfeldt 

This is patch **version bump** because it fixes a non-concordance with the current orderform
